### ISSUE TITLE
The meaning of num_feature_levels.

### ISSUE THREAD

**Issue by TsingWei** [IDEA-opensource/DAB-DETR#20]
Thanks for your great and detailed code!

In `dab_deformable_detr.py`, is this part about dealing with the insufficient channel of features (for Deformable DETR)?

> https://github.com/IDEA-opensource/DAB-DETR/blob/309f6ad92af7a62d7732c1bdf1e0c7a69a7bdaef/models/dab_deformable_detr/dab_deformable_detr.py#L169-L181

It seems that the last channel of features is put to `backbone[1]` repeatedly. (channel? or something else?)
Could you please help me understand that?

**Comment by SlongLiu** [comment]
Yes, it is. By default, ```num_feature_levels=4``` but ```3``` different scales are extracted from the backbone. Hence it will 2x upsample the highest feature map(C4) as the 4th feature level.
